### PR TITLE
Feature/cfa 340 rename table to better naming

### PR DESCRIPTION
**Pull request by maxime-poulain**
This PR gives more meaningful names to CFA's tables.

Generated sql script from the database:
[script.txt](https://github.com/smartcoop/smart-cfa/files/8547479/script.txt)

image

### CODE REVIEW

File: src/UserAdmin/src/Smart.FA.Catalog.Infrastructure/Persistence/Read/TrainingQueries.cs [hunk 1]
```diff
@@ -20,12 +20,12 @@ public async Task<TrainingDto> FindAsync(int trainingId, string language, Cancel
     {
         var sql = @"SELECT
 	                    T.Id 'TrainingId',
-	                    T.StatusId,
+	                    T.TrainingStatusId,
```
Comment (Victor-van-Duijnen) [inline src/UserAdmin/src/Smart.FA.Catalog.Infrastructure/Persistence/Read/TrainingQueries.cs:23]: Just noticed it should now be changed to StatusTypeId here.

File: src/UserAdmin/src/Smart.FA.Catalog.Infrastructure/Persistence/Read/TrainingQueries.cs [hunk 2]
```diff
@@ -69,24 +69,24 @@ int trainerId
     {
         var sql = @"SELECT
 	                    T.Id 'TrainingId',
-	                    T.StatusId,
+	                    T.StatusTypeId,
 	                    TD.Title,
 	                    TD.Goal,
                         TD.Language,
-                        TC.TrainingTopicId 'TopicId'
+                        TC.TopicId 'TopicId'
```
Comment (Victor-van-Duijnen) [inline src/UserAdmin/src/Smart.FA.Catalog.Infrastructure/Persistence/Read/TrainingQueries.cs:76]: Same comment as above, alias is useless.

File: src/UserAdmin/src/Smart.FA.Catalog.Infrastructure/Persistence/Read/TrainingQueries.cs [hunk 3]
```diff
@@ -40,15 +40,15 @@ int trainerId
     {
         var sql = @"SELECT
 	                    T.Id 'TrainingId',
-	                    T.StatusId,
+	                    T.TrainingStatusId,
```
Comment (Victor-van-Duijnen) [inline src/UserAdmin/src/Smart.FA.Catalog.Infrastructure/Persistence/Read/TrainingQueries.cs:43]: Same as above, should be renamed to StatusTypeId

**Comment by Victor-van-Duijnen** [comment]
I have allowed myself to make the small fixes mentionned in the comments so that that PR is blocked the least amount of time since it impacts many files (and would create conflicts if left too long in review)

**Review by Victor-van-Duijnen — APPROVED** [review]

**Comment by maxime-poulain** [comment]
We need to be aligned with the showcase team before merging that.

**Pull request closed without merging**